src/main/java/com/elasticbox/jenkins/model/profile/PolicyProfileType.java

```diff
@@ -37,6 +43,12 @@ public ProviderType provider() {
             return ProviderType.AZURE;
         }
     },
+    AZURE_ARM("/azure-arm/arm/profile") {
+        @Override
+        public ProviderType provider() {
+            return ProviderType.AZURE;
```

Comment:
@gsanchezu On ElasticBox ```Azure``` and ```AzureARM``` are two different provider types. Not sure if you should also define both here.